Unbreak publishing

The changes in #27 unfortunately broke publishing, as can be seen in [the build log](https://github.com/perlun/perlang/commit/766eedfd9ec80f04624fb762b3af8c2f4dd9879b/checks?check_suite_id=467691201):

> ##[error]/opt/hostedtoolcache/dncs/3.1.101/x64/sdk/3.1.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Publish.targets(270,5): error NETSDK1095: Optimizing assemblies for performance is not supported for the selected target platform or architecture. Please verify you are using a supported runtime identifier, or set the PublishReadyToRun property to false. [/home/runner/work/perlang/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj]
> ##[error]Process completed with exit code 1.

This is when trying to build for `osx-x64`. We'd need to find a way to do one of the following:

1. Disable `PublishReadyToRun` for these builds
2. Skip macOS builds for now
3. Revert #27 (i.e. don't use R2R on any platform, even where it's supported)

--- a/.github/workflows/publish-packages.yml
+++ b/.github/workflows/publish-packages.yml
@@ -24,7 +24,7 @@ jobs:
     # Build releases
     #
     - name: Build linux-x64 release
-      run: dotnet publish -c Release -r linux-x64 --self-contained true
+      run: dotnet publish -c Release -r linux-x64 --self-contained true /p:PublishReadyToRun=true
       working-directory: ./Perlang.ConsoleApp
 
     - name: Build osx-x64 release

--- a/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj
+++ b/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj
@@ -4,7 +4,8 @@
         <OutputType>Exe</OutputType>
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <PublishTrimmed>true</PublishTrimmed>
-        <PublishReadyToRun>true</PublishReadyToRun>
+        <!-- PublishReadyToRun is set on command line since it cannot
+             run in cross-compilation scenarios -->
         <AssemblyName>perlang</AssemblyName>
     </PropertyGroup>
 

